src/container_registry/docr.rs

```diff
@@ -305,7 +309,7 @@ impl ContainerRegistry for DOCR {
     fn push(&self, image: &Image, force_push: bool) -> Result<PushResult, EngineError> {
         let registry_name = self.get_registry_name(image)?;
 
-        let _ = match self.create_repository(&image) {
+        let _ = match self.create_repository(image) {
```

Comment:
Is `let _ =` still needed in this context?

Reply:
Was wondering the same thing. Maybe not? 🤔

Reply:
Will patch 👍